app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt

```diff
@@ -742,12 +742,12 @@ class AdminPinActivityTest {
       )
     ).use {
       onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
-        typeText("12345"),
+        editTextInputAction.appendText("12345"),
         closeSoftKeyboard()
       )
       onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_confirm_pin)))).perform(
         scrollTo(),
-        typeText("54321"),
+          editTextInputAction.appendText("54321"),
```

Comment:
Please fix the formatting in this PR as tests lint tests are failing.
<img width="975" alt="Screenshot 2020-10-15 at 4 19 43 AM" src="https://user-images.githubusercontent.com/9396084/96053760-c86a5c00-0e9d-11eb-93b7-0d07e48fe02d.png">

Reply:
Will do...For some reason I couldn't see the results before - definitely helps to see why it's failing....Will resubmit shortly.

Reply:
Check our wiki page here for ktlint setup and fixes - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide

Reply:
Done

Reply:
@bladebunny Please do not resolve comments. The person who has started the comment thread should resolve the comment thread. In this case I had started the comment thread and therefore I was suppose to resolve this thread. Let's follow this in future PRs. Thanks.